enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/søknad/Søknadsfelt.kt

```diff
@@ -1,4 +1,3 @@
 package no.nav.familie.kontrakter.ef.søknad
 
-data class Søknadsfelt<T>(val label: String,
-                          val verdi: T)
+data class Søknadsfelt<T>(val label: String, val verdi: T, val alternativer: List<String>? = null)
```

Comment:
Er det mulig å bruke emptylist som default, så trenger vi ikke nullsjekk?

Reply:
Noen valg har vel 8 ulike lange tekster, det blir veldig mye data som sendes inn?

Reply:
Jepp - blir ofte mye tekst. Bruker så liten font som mulig i pdf mal og det gjør jo en allerede lang pdf(søkand) enda lenger. Vi velger likevel å ta med dette for dokumentasjon og kontekst til saksbehandler. 

emtylist - jeg sjekker :) Må se hvordan det ser ut i mottak.

Reply:
Ønsker saksbehandlere å se alle ulike alternativ? Det blir vel en svær PDF? 
Og det vil vel ikke bli vist i saksbehandling senere ettersom vi kun lagrer verdiene og ikke label (?)

Reply:
Label er jo felles for alle alternativene, @blommish ? Så det er value for de "andre" valgene som gjør at saksbehandler kan vurdere "hva" bruker har svart.

Reply:
Jepp. De har faktisk spurt om det. Litt for å slippe slike situasjoner: 
Gjelder dette deg? 
Nei
Som ikke gir så mye mening. 

Vi så på flere muligheter, og lengde på pdf var absolutt noe som gjorde at vi ikke hadde lyst til å gå for denne. 
Her er favro-oppgaven hvor litt av diskusjonen vi har hatt kan leses mellom linjene. https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=Tea-2509

Når det gjelder emptyList har jeg testet dette og - det fungerer bedre også i mottak :) Endrer denne.

Reply:
Men nå vil jo "normalsituasjon" være at det ikke er noen alternativer på et søknadsfelt. Blir det mange unødvendige lister? Alle nullsjekkene mine i mottak ble jo bare byttet ut med isNotEmpty().

Reply:
@charliemidtlyng Stemmer det ja! :D